pandas-stubs/core/accessor.pyi

```diff
@@ -1,8 +1,5 @@
 from typing import Any
 
-class DirNamesMixin:
-    def __dir__(self): ...
```

Comment:
PandasObject and this class were causing pyright's MRO issue. I believe these two classes are not public and they provide no typing benefit (their methods are already provided by object).